src/backend/distributed/planner/multi_router_planner.c

```diff
@@ -534,6 +536,18 @@ ModifyQuerySupported(Query *queryTree, Query *originalQuery, bool multiShardQuer
 	Node *onConflictWhere = NULL;
 	CmdType commandType = queryTree->commandType;
 
+	RelationRestrictionContext *relationRestrictionContext =
```

Comment:
Ops, this fails to catch `VIEW`s because they don't appear in the restriction context.

So, using a VIEW instead of local table fails now.